amq/src/test/java/org/jboss/test/arquillian/ce/amq/AmqPersistentSecuredTest.java

```diff
@@ -65,10 +66,15 @@
         @TemplateParameter(name = "AMQ_KEYSTORE_PASSWORD", value = "password")})
 @RoleBinding(roleRefName = "view", userName = "system:serviceaccount:${kubernetes.namespace}:default")
 @OpenShiftResources({
-    @OpenShiftResource("classpath:amq-app-secret.json"),
+    @OpenShiftResource("https://raw.githubusercontent.com/jboss-openshift/application-templates/master/secrets/amq-app-secret.json"),
```

Comment:
Use the "overridable" url, where users can override it. Like https://github.com/jboss-openshift/ce-testsuite/blob/master/eap/eap70/src/test/java/org/jboss/test/arquillian/ce/eap70/Eap70BasicTest.java#L35